---
permalink: /research/
title: "Research"
author_profile: true
---

## Experience

|             |                    |                                                        |                | 
| ----------- | ------------------ | ------------------------------------------------------ | -------------- | 
| 2023 - present | Research Associate | Bigelow Laboratory for Ocean Sciences | East Boothbay, ME |
| 2023 - present | Research Associate | Colby College | Waterville, ME | 
| 2020 - 2023 | Research Technician | Bigelow Laboratory for Ocean Sciences | East Boothbay, ME |
| 2019 | Research Associate | [Commercial Fisheries Research Foundation](http://www.cfrfoundation.org/) | Saunderstown, RI |
| 2018 - 2019 | Research Fellow | [Richard H Goodwin Environmental Research Fellowship](https://www.conncoll.edu/goodwin-fellowship/) | New London, CT |
| Summer 2018 | Intern | NOAA Southwest Fisheries Science Center | La Jolla, CA | 
| Summer 2017 | Student Researcher | Connecticut College Summer Research in the Sciences | New London, CT |
| Summer 2016 | Student Researcher | Connecticut College Summer Research in the Sciences | New London, CT |
| Summer 2014 | Volunteer | NOAA Southwest Fisheries Science Center | La Jolla, CA |

## Research Interests

* Fisheries Biology
* Highly Migratory Species
* Research Software Development
* Ocean Forecasting
* Knowledge Co-production

## Professional Affiliations

[Coastal and Esturaine Research Federation](https://www.cerf.science/)

[Association for the Sciences of Limnology and Oceanography](https://www.aslo.org/)

[Ecological Forecasting Initiative](https://ecoforecast.org/)


## Selected Projects 

### Maine Shellfish Biotoxins

### Blue Mussel Spat Settlement in Coastal Maine

### Assessing Marine Protected Area Success with OBIS Data

The Ocean Biodiversity Information System (OBIS) provides a global repository of marine species occurrence records. Using boundaries of marine protected areas (MPAs) around the world, species records were queried from OBIS for each MPA and used to calculate a biodiversity metric over 20 year time increments. Some of the guiding questions were: Is biodiversity in an MPA related to its size, age or distance from shore? The project team was formed during OceanHackWeek 2021 and continued to meet regularly for a year following the event. The analysis and tools developed for the project are located in this [repo](https://github.com/BigelowLab/mpatools).

### *Seriola dorsalis* Aquaculture Genomics

As a candidate species for aquaculture in Southern California, genomic resources are necessary to accelerate the development of Seriola dorsalis aquaculture. With knowledge of genetic basis behind locally adapted traits between wild populations of important aquaculture species, individuals that possess these valuable phenotypes can be used for selective breeding in aquaculture. In this study, a population genomic tool, Bayescan version 2.1, was implemented on a dataset of 76,974 loci of SNPs found within the genome of S. dorsalis, in order to search for loci being selected for based on allele frequencies between the three populations. No loci were found to have a probability of being naturally selected for higher than 11.042%, which does not provide any evidence of locally adapted, population specific traits that could be exploited through selective breeding in aquaculture. Based on these results, any fish taken from the region surrounding the three sample locations would possess similar traits, and the three populations can be managed as one.

### Fish Stock Genetic Diversity

Many current fishery stock assessment methods strongly rely on the amount of fish harvest reported at the dock by fishermen. We seek a method for fish stock assessment that is based on transcriptome measures. In this study we were interested in the correlation between transcriptome level diversity and changes in the phenotype expression ability of commercially targeted fish. By analyzing the complexity of miRNA/RNAi 7mer binding sites in the 3’UTR regions, inferences are made as to the accessible repertoire of phenotypes for the organism. If fewer phenotypes are available, for use in response to environmental change, or for use in extending habitable niche, such as by ‘schooling’, then significant loss of fishery stock may result. Preliminary results indicate Gadus Morhua (Atlantic Cod) has undergone such a loss in transcript regulatory complexity, which appears to be associated with the collapse of the Cod fishery in the Gulf of Maine.

